### ISSUE TITLE
Intermittent connectivity issues between `gateway` and on-premise devices

### ISSUE THREAD

**Issue by ethnt** [ethnt/e10#95]

**Comment by ethnt** [comment]
A few different things in the Tailscale logs.

**DNS issues**

```
Dec 23 22:27:42 htpc tailscaled[1695]: [RATELIMIT] format("dns: resolver: forward: no upstream resolvers set, returning SERVFAIL") (11 dropped)
Dec 23 22:27:42 htpc tailscaled[1695]: dns: resolver: forward: no upstream resolvers set, returning SERVFAIL
Dec 23 22:27:42 htpc tailscaled[1695]: dns: DNS resolution failed due to missing upstream nameservers.  Recompiling DNS configuration.
Dec 23 22:27:42 htpc tailscaled[1695]: dns: Set: {DefaultResolvers:[] Routes:{tailc3772.ts.net.:[] ts.net.:[199.247.155.53 2620:111:8007::53]}+65arpa SearchDomains:[tailc3772.ts.net.] Hosts:20}
Dec 23 22:27:42 htpc tailscaled[1695]: dns: Resolvercfg: {Routes:{ts.net.:[199.247.155.53 2620:111:8007::53]} Hosts:20 LocalDomains:[tailc3772.ts.net.]+65arpa}
Dec 23 22:27:42 htpc tailscaled[1695]: dns: OScfg: {Nameservers:[100.100.100.100] SearchDomains:[tailc3772.ts.net.] MatchDomains:[tailc3772.ts.net. ts.net.]+65arpa}
Dec 23 22:27:42 htpc tailscaled[1695]: dns: resolver: forward: no upstream resolvers set, returning SERVFAIL
Dec 23 22:27:42 htpc tailscaled[1695]: dns: resolver: forward: no upstream resolvers set, returning SERVFAIL
Dec 23 22:27:42 htpc tailscaled[1695]: dns: resolver: forward: no upstream resolvers set, returning SERVFAIL
Dec 23 22:27:42 htpc tailscaled[1695]: dns: resolver: forward: no upstream resolvers set, returning SERVFAIL
Dec 23 22:27:42 htpc tailscaled[1695]: [RATELIMIT] format("dns: resolver: forward: no upstream resolvers set, returning SERVFAIL")
```

**DERP**

```
Dec 23 22:32:04 htpc tailscaled[1695]: control: NetInfo: NetInfo{varies=false hairpin= ipv6=false ipv6os=true udp=true icmpv4=false derp=#1 portmap=active-UMC link="" firewallmode="ipt-defa>
Dec 23 22:32:01 htpc tailscaled[1695]: magicsock: disco: failed to send call-me-maybe to 127.3.3.40:1: too many DERP packets queued; dropping
Dec 23 22:31:55 htpc tailscaled[1695]: open-conn-track: timeout opening (TCP 100.71.71.64:49672 => 100.73.138.17:3100) to node [eDpkc]; online=yes, lastRecv=8s
Dec 23 22:31:54 htpc tailscaled[1695]: open-conn-track: timeout opening (TCP 100.71.71.64:43020 => 100.73.138.17:3100) to node [eDpkc]; online=yes, lastRecv=8s
Dec 23 22:31:48 htpc tailscaled[1695]: open-conn-track: timeout opening (TCP 100.71.71.64:43020 => 100.73.138.17:3100) to node [eDpkc]; online=yes, lastRecv=2s
Dec 23 22:31:43 htpc tailscaled[1695]: open-conn-track: timeout opening (TCP 100.71.71.64:43020 => 100.73.138.17:3100) to node [eDpkc]; online=yes, lastRecv=9s
Dec 23 22:31:40 htpc tailscaled[1695]: control: NetInfo: NetInfo{varies= hairpin= ipv6=false ipv6os=true udp=false icmpv4=true derp=#1 portmap=active-UMC link="" firewallmode="ipt-default"}
Dec 23 22:31:38 htpc tailscaled[1695]: netcheck: UDP is blocked, trying ICMP
Dec 23 22:31:38 htpc tailscaled[1695]: netcheck: netcheck: UDP is blocked, trying HTTPS
Dec 23 22:31:35 htpc tailscaled[1695]: control: NetInfo: NetInfo{varies= hairpin= ipv6=false ipv6os=true udp=false icmpv4=false derp=#1 portmap=active-UMC link="" firewallmode="ipt-default"}
Dec 23 22:31:33 htpc tailscaled[1695]: netcheck: UDP is blocked, trying ICMP
Dec 23 22:31:33 htpc tailscaled[1695]: netcheck: netcheck: UDP is blocked, trying HTTPS
```

**Comment by ethnt** [comment]
Closed due to #96
